client/layout/style.scss

```diff
@@ -1,10 +1,52 @@
-.layout {
-	&.is-section-preview {
-		height: 100%;
-		overflow: hidden;
+/*
+	Layout Elements
+	.layout__loading - Displays when loading Claypso
+	.layout__content - Contains primary and secondary elements
+		.layout__primary - Where the main content lives
+		.layout__secondary - Contains the site selector and sidebar elements
+			.sidebar
+			.site-selector
+*/
+
+// Set transitons for layout elements
+.layout__primary,
+.layout__secondary,
+.layout__secondary .sidebar,
+.layout__secondary .site-selector {
+	transition: transform 0.2s cubic-bezier(0.770, 0.000, 0.175, 1.000),
+				opacity 0.2s ease-in-out;
+}
+
+// If things take a bit to load...
+.layout__loader {
+	border-bottom: 1px solid var( --masterbar-border-color );
+	height: 46px;
+	margin-left: -10%;
+	position: absolute;
+		left: 50%;
+		top: 0;
+	width: 20%;
+	z-index: z-index( 'root', '.layout__loader' );
+
+	// set a delay threshold for opacity changes
+	// prevents showing loader on fast connections
+	visibility: hidden;
```

Comment:
Again is there a reason for not using `display: none`? Does the delay not work otherwise? I assume this is for the animation, but I'm thinking this might work if we add a` display:block` on the `is-active` modifier

Reply:
This is the one part of the file I didn't touch! I'm not sure why its using `visibility` instead of `display`. I'll update it.